app/static/voice.js

```diff
@@ -0,0 +1,68 @@
+function listVoices(synth, voiceSelect) {
+	voices = synth.getVoices();
```

Comment:
Prefer indentation using 2 whitespaces (max 4, but not 8)

Reply:
I don't see 8... but checked it again :)